infrastructure/application/index.ts

```diff
@@ -52,6 +52,10 @@ const CUSTOM_DOMAINS =
           domain: "planningservices.newcastle.gov.uk",
           name: "newcastle",
         },
+        {
+          domain: "stalbans.newcastle.gov.uk",
```

Comment:
Is this right or a copy and paste error?

Reply:
🤦

Reply:
Thank you very much! Will fix now